sklearn/ensemble/tests/test_forest.py

```diff
@@ -947,6 +947,13 @@ def check_class_weights(name):
     clf2.fit(iris.data, iris.target, sample_weight)
     assert_almost_equal(clf1.feature_importances_, clf2.feature_importances_)
 
+    # When sample_weights is an unsupported array type, it checks if it raises
+    # an exception. With Python 2.x lists it complains there is no copy()
```

Comment:
This comment is a bit odd as not it doesn't fail. Maybe say "used to fail on python2 for lists" or something. This checks if it passes, not if it raises.